guides/v2.2/cloud/architecture/architecture-scaling.md

```diff
@@ -0,0 +1,54 @@
+---
+group: cloud-guide
+title: Scaling architecture
+functional_areas:
+  - Cloud
+---
+
+The Cloud infrastructure scales according to your resource needs to achieve greater efficiency. The {{site.data.var.ece}} auto-scaling monitors your applications and automatically adjusts capacity to maintain steady, predictable performance. Converting to this architecture helps to mitigate problems, such as latency, large spikes in traffic, or running out of storage.
```

Comment:
No `auto-scaling`. It is scripted and we monitor when we need to initiate it but it's not an auto-scaling.